src/ConnectedMode/UI/Credentials/CredentialsWnd.xaml

```diff
@@ -45,7 +45,7 @@
             </Grid.ColumnDefinitions>
 
             <TextBlock Grid.Column="0" Text="{x:Static res:UiResources.AuthenticationDescription}" VerticalAlignment="Center" />
-            <TextBlock Grid.Column="1" Text="{Binding Path=Connection.Id}" VerticalAlignment="Center" Margin="5,0" FontWeight="Bold"/>
+            <ui:ConnectionInfoComponent Grid.Column="1" ConnectionInfo="{Binding Path=Connection}"/>
```

Comment:
Minor: I would remove the `:` here to make it more fluent
![image](https://github.com/user-attachments/assets/095135fa-8fa4-41d8-b967-b82d2db785f5)